packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss

```diff
@@ -14,6 +14,25 @@ $ms-choiceField-transition-timing: cubic-bezier(.4, 0, .23, 1);
 //== Component: Choicefield group
 //
 
+@mixin highContrastBorder($color) {
```

Comment:
Aren't we using this mixin elsewhere? If so, you should import it and use it instead. If we AREN'T, we probably should. I.E. Look at how this is solved in other areas.